xml/decoders/esu/v5fnOutCVs.xml

```diff
@@ -51,7 +51,7 @@
         <decVal/>
         <label xml:lang="de">Ausgang automatisch Ausschalten</label>
     </variable>
-    <variable label="Brightness CV" CV="16.0.262" default="31" comment="Dummy to work around sheet operations/qualifiers issue">
+    <variable item="Brightness CV262" label="Brightness CV" CV="16.0.262" default="31" comment="Dummy to work around sheet operations/qualifiers issue">
```

Comment:
I see no reason why the `label="xxx"`can't go. Leaving it there is pointless.

The new `item` is actually a dummy variable attached to a CV rather than a dummy CV. I wonder if we could reflect that in the `item` name.

Reply:
The PR is not concerned with the labels (might break, up to author) but making sure unique item attribute is in place. Also didn't check if the item could have a function in the qualifier "issue" described in the var comment.